.storybook/stories/KitchenSink/KitchenSink.stories.tsx

```diff
@@ -97,7 +97,10 @@ import { GroupedFields } from '../../../packages/react/src/grouped-fields';
 import { TimeInput } from '../../../packages/react/src/time-input';
 import { TimePicker } from '../../../packages/react/src/time-picker';
 import { useTernaryState } from '../../../packages/react/src/core';
-import { COUNTRY_OPTIONS } from '../../../docs/components/designSystemComponents';
+import {
+	COUNTRY_OPTIONS,
+	FeatureLinkList,
```

Comment:
The auto import has imported it from the wrong place here. It should be imported from its package